src/assets/stylesheets/scrivitoSpaceWidget.scss

```
@@ -0,0 +1,81 @@
+hr.scrivito-space-widget {
+  border: 0;
+  margin: 0;
+  clear: both;
+}
+
+hr.scrivito-space-widget.xs-small {
+  padding: 5px 0;
```

```suggestion
  padding-top: 5px;
```

This suggestion applies to all given sizes.